src/backend/src/services/change-requests.services.ts

```diff
@@ -578,8 +579,9 @@ export default class ChangeRequestsService {
       }
     });
 
-    if (!wbsElement) throw new NotFoundException('WBS Element', `${carNumber}.${projectNumber}.${workPackageNumber}`);
-    if (wbsElement.dateDeleted)
+    if (!projectProposedChanges && !wbsElement)
+      throw new NotFoundException('WBS Element', `${carNumber}.${projectNumber}.${workPackageNumber}`);
+    if (wbsElement && wbsElement.dateDeleted)
```

Comment:
i mean u dont really need first wbsElement check, you can just say if wbsElement?.dateDeleted

Reply:
Yeah I was just messing around with it because there doesn't seem to be a good way to do wbsNumbers